### DISCUSSION TITLE
Human Skin Scan

### DISCUSSION THREAD

**Discussion by shivasnape** [sceneview/sceneview#480] [Ideas]
Can this AR Core / Scene form library be used for scanning human hand or leg to see the scratches?

**Comment by ThomasGorisse** [comment]
We are open to any PR on that.
Feel free to create one and we will help you with pleasure on it.

**Comment by ThomasGorisse** [comment]
SceneView is a 3D/AR rendering library, not a scanning or photogrammetry tool. For human skin/body scanning, you'd need:

1. **Capture**: Use ARCore depth API + camera frames (accessible via `ARSceneView`'s `onSessionUpdated`)
2. **Reconstruction**: Process depth + RGB data with a 3D reconstruction library (e.g. Open3D, or a custom pipeline)
3. **Visualization**: Display the resulting mesh in SceneView

SceneView can help with step 3 (rendering the scanned mesh as a GLB), and provide the camera/depth data for step 1, but the actual scanning/reconstruction logic would need to come from a separate library or ML pipeline.
